tests/kernels/test_moe.py

```diff
@@ -51,10 +49,10 @@ def test_fused_moe(
     w2 = torch.randn((e, k, n), device='cuda', dtype=dtype) / 10
 
     score = torch.randn((m, e), device='cuda', dtype=dtype)
-    score = torch.softmax(score, dim=-1)
-    topk_weight, topk_ids = torch.topk(score, topk)
+    triton_output = fused_moe(a, w1, w2, score, topk, renormalize=False)
 
-    triton_output = fused_moe(a, w1, w2, topk_weight, topk_ids, False)
+    score = torch.softmax(score, dim=-1, dtype=torch.float32)
+    topk_weight, topk_ids = torch.topk(score, topk)
     torch_output = torch_moe(a, w1, w2, topk_weight, topk_ids)
```

Comment:
One more thing I noticed: It might be worth pushing the
```
    score = torch.softmax(score, dim=-1, dtype=torch.float32)
    topk_weight, topk_ids = torch.topk(score, topk)
```
code into `torch_moe` so that `fused_moe` and `torch_moe` keep having the same signature and behavior :)

Reply:
Good point! Fixed.